modules/policy-change-management.adoc

```diff
@@ -22,6 +22,22 @@ Change history is captured in the *Cluster History* section in the OpenShift Clu
 * Scaling the cluster persistent storage
 * Upgrading the cluster
 
+You can implement a maintenance exclusion by avoiding changes in {cluster-manager} for the following components:
+
+* Deleting a cluster
+* Adding, modifying or removing identity providers
```

Comment:
Change to: Adding, modifying, or removing identity providers
Per  the ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series